PhysicsTools/NanoAOD/python/nano_cff.py

```diff
@@ -178,9 +180,33 @@ def nanoAOD_addDeepFlavourTagFor94X2016(process):
     return process
 
 
+def nanoAOD_addDeepBoostedJetForPre103X(process):
+    print "Updating process to run DeepBoostedJet on datasets before 103X"
+    from RecoBTag.MXNet.pfDeepBoostedJet_cff import _pfDeepBoostedJetTagsAll as pfDeepBoostedJetTagsAll
+    updateJetCollection(
+       process,
+       jetSource = cms.InputTag('slimmedJetsAK8'),
+       pvSource = cms.InputTag('offlineSlimmedPrimaryVertices'),
+       svSource = cms.InputTag('slimmedSecondaryVertices'),
+       rParam = 0.8,
+       jetCorrections = ('AK8PFPuppi', cms.vstring(['L2Relative', 'L3Absolute', 'L2L3Residual']), 'None'),
+       btagDiscriminators = pfDeepBoostedJetTagsAll,
+       postfix='AK8Puppi',
+       printWarning = False
+       )
+    process.looseJetIdAK8.src = "selectedUpdatedPatJetsAK8Puppi"
+    process.tightJetIdAK8.src = "selectedUpdatedPatJetsAK8Puppi"
+    process.tightJetIdLepVetoAK8.src = "selectedUpdatedPatJetsAK8Puppi"
+    process.slimmedJetsAK8WithUserData.src = "selectedUpdatedPatJetsAK8Puppi"
+    return process
+
+
 def nanoAOD_customizeCommon(process):
     run2_miniAOD_80XLegacy.toModify(process, nanoAOD_addDeepBTagFor80X)
     run2_nanoAOD_94X2016.toModify(process, nanoAOD_addDeepFlavourTagFor94X2016)
+    for modifier in run2_nanoAOD_94X2016, run2_nanoAOD_94XMiniAODv1, run2_nanoAOD_94XMiniAODv2:
```

Comment:
This leads to updateJetCollection being called more than once in the same job from other customize functions, leading to possible inconsistencies.
We need to change the model of these customize functions.

Reply:
Correcting myself, for the moment, this updateJetCollection is called on a different collection and with postfix, so for the moment it's ok.